aea/cli/__main__.py

```diff
@@ -19,6 +19,8 @@
 #
 # ------------------------------------------------------------------------------
 
+# pragma: nocover  # noqa: E800
```

Comment:
let's not use `pragma: no cover`, it does us no good rather have "less coverage" as long as it's actual true coverage (also [Trello ticket](https://trello.com/c/8G17Vn54/894-remove-pragma-nocover-from-repositories) exists to exterminate the usage of `pragma: no cover`